app/internal/shabbat.py

```diff
@@ -0,0 +1,41 @@
+from typing import Dict
+from datetime import datetime
+
+
+def shabbat_time_by_user_location(shabbat_time: Dict[str, str]) -> Dict:
+    """Returns the shabbat time of the user location..
+
+        Args:
+            Shabbat details.
+
+        Returns:
+            Shabbat start end ending time.
+        """
+    shabbat_limit = {
+        'start_hour': shabbat_time['items'][5]['title'].split(': ')[1],
+        'start_date': datetime.strptime(
+            shabbat_time['items'][5]['date'].split('T')[0], "%Y-%m-%d"
+        ).date(),
+        'end_hour': shabbat_time['items'][7]['title'].split(': ')[1],
+        'end_date': datetime.strptime(
+            shabbat_time['items'][7]['date'].split('T')[0], "%Y-%m-%d"
+        ).date()}
+    return shabbat_limit
+
+
+def get_shabbat_if_date_friday(
+        shabbat_time: Dict[str, str],
+        date: datetime) -> Dict:
```

Comment:
Split before ")" and return Dict of what?

Reply:
i edit